util.js

```diff
@@ -23,13 +24,28 @@ function spawn (cmd, args, cb) {
   })
 }
 
+function fork (file, cb) {
+  return cp.fork(file).on('exit', function (code) {
+    if (code === 0) return cb()
+    cb(error.spawnFailed(file, code))
+  })
+}
+
 function exec (cmd, cb) {
   return execSpawn(cmd, {stdio: 'inherit'}).on('exit', function (code) {
     if (code === 0) return cb()
     cb(error.spawnFailed(cmd, [], code))
   })
 }
 
+function run (item, cb) {
+  if (item.substr(-3) === '.js' && fs.existsSync(item)) {
```

Comment:
Why check for file existence?

Reply:
Also, prefer `path.extname(item) === '.js'`

Reply:
Why the `.fork()` in the first place? Doesn't `.exec()` work?